docs/2_app_operators/1_packaging_an_app.md

```diff
@@ -0,0 +1,185 @@
+# Packaging an Application
```

Comment:
I think I may have misinterpreted the `app operator` persona. Does this seem like something that should instead fall under `developer` persona? @prasek @negz